Maddori.Apple/Maddori.Apple/Screen/Home/AddFeedback/AddFeedbackContentViewController.swift

```diff
@@ -0,0 +1,276 @@
+        case .fourth:
+            return UIImageView(image: ImageLiterals.imgProgress4)
+        }
+    }()
+    private lazy var currentStepLabel: UILabel = {
+        let label = UILabel()
+        switch step {
+        case .second:
+            label.text = TextLiteral.addFeedbackContentViewControllerCurrentStepLabel2
+        case .third:
+            label.text = TextLiteral.addFeedbackContentViewControllerCurrentStepLabel3
+        case .fourth:
+            label.text = TextLiteral.addFeedbackContentViewControllerCurrentStepLabel4
+        }
+        label.textColor = .black100
+        label.font = .title2
+        label.setLineSpacing(to: 4)
+        label.numberOfLines = 2
+        return label
+    }()
+    private lazy var currentStepDescriptionLabel: UILabel = {
+        let label = UILabel()
+        switch step {
+        case .second:
+            label.text = TextLiteral.addFeedbackContentViewControllerCurrentStepDescriptionLabel2
+        case .third:
+            label.text = TextLiteral.addFeedbackContentViewControllerCurrentStepDescriptionLabel3
+        case .fourth:
+            label.text = TextLiteral.addFeedbackContentViewControllerCurrentStepDescriptionLabel4
+        default:
+            label.text = ""
+        }
+        label.textColor = .gray400
+        label.font = .body2
+        label.setLineSpacing(to: 4)
+        label.numberOfLines = 0
+        return label
+    }()
+    private let dividerView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .gray300
+        return view
+    }()
+    private let feedbackContentTextView: UITextView = {
+        let view = UITextView()
+        view.backgroundColor = .white200
+        view.font = .body1
+        return view
+    }()
+    private let doneButtonView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .white200
+        return view
+    }()
+    private lazy var doneButton: MainButton = {
+        let button = MainButton()
+        button.title = TextLiteral.addFeedbackContentViewControllerButtonNext
+        button.isDisabled = true
+        let action = UIAction { [weak self] _ in
+            self?.didTappedDoneButton()
+        }
+        button.addAction(action, for: .touchUpInside)
+        return button
+    }()
+    
+    // MARK: - life cycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupDelegate()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        setupNotificationCenter()
+        feedbackContentTextView.becomeFirstResponder()
+        
+        contentString = contentString.replacingOccurrences(of: "\(currentStepString)|\n\n\(currentStepString)", with: "", options: .regularExpression)
+    }
+    
+    override func render() {
+        view.addSubview(progressImageView)
+        progressImageView.snp.makeConstraints {
+            $0.top.equalTo(view.safeAreaLayoutGuide.snp.top).inset(8)
```

Comment:
p3;
```suggestion
            $0.top.equalTo(view.safeAreaLayoutGuide.snp.top).inset(Size.topPadding)
```
위에서 만들어둔 Size enum.. 이곳에 사용하시려고 하셨죠..? 🧐

Reply:
[28604a7](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/211/commits/28604a7b47746268fb45df11a52b7ce588d6c8a1)
와.... 작성자의 의도 파악 능력 무엇...
수능에서도 출제자의 의도 파악 잘 하셨을 것 같습니다...

Reply:
역시 Ewha..

Reply:
역시 ㄸ좀..?

Reply:
예..??